README.md

```diff
@@ -22,9 +22,28 @@ Then navigate to the `topicalguide` directory and optionally switch over to the
 
 ### 2. Install Dependencies
 
+Superuser permissions are necessary to install the Topical Guide as shown in the current instructions.
```

Comment:
Since I don't really understand the virtual environment thing or how to use
pyvenv or whatever, would it be okay if we just used my README for now and
then someone else can make the changes they want on their own forks and
submit separate pull requests? If not, someone can explicitly tell me new
instructions to put in the README and I will. I'm just pretty ignorant
about installing things and permissions and environments and all of that.

I will let you know when I've made all of the other changes.

On Wed, Feb 10, 2016 at 12:15 PM, Madison Nuss <madisonnuss@gmail.com>
wrote:

> Sorry, I meant that Nozomu gave me that explanation of why we need to use
> a virtual environment. He didn't tell me to put it in the README. I don't
> mean to put blame on anyone.
>
> On Wed, Feb 10, 2016 at 12:09 PM, Madison Nuss <madisonnuss@gmail.com>
> wrote:
>
>> On the next line, it says "In order to circumvent the need for superuser
>> permissions, ..." I didn't want to put the first line, but Nozomu told me
>> to. I'll get rid of it.
>>
>> I still use virtualenv because that's what I was told to use, and I put
>> the link to the documentation for pyenv below. I don't know how to use
>> pyenv so I didn't put detailed instructions for it. Someone else can change
>> that if they want.
>>
>> On Wed, Feb 10, 2016 at 9:13 AM, Connor Cook <notifications@github.com>
>> wrote:
>>
>>> In README.md
>>> <https://github.com/BYU-NLP-Lab/topicalguide/pull/87#discussion_r52480160>
>>> :
>>>
>>> > @@ -22,9 +22,28 @@ Then navigate to the `topicalguide` directory and optionally switch over to the
>>> >
>>> >  ### 2. Install Dependencies
>>> >
>>> > +Superuser permissions are necessary to install the Topical Guide as shown in the current instructions.
>>>
>>> Maybe here we should just say something about how we are using
>>> virtualenv to avoid the need for superuser permissions? It sounds like we
>>> are telling the user that you actually need superuser instructions to do
>>> what is talked about below.
>>>
>>> Also, do we still use virtualenv? I think Python3 uses pyvenv, or
>>> something like that, instead.
>>>
>>> —
>>> Reply to this email directly or view it on GitHub
>>> <https://github.com/BYU-NLP-Lab/topicalguide/pull/87/files#r52480160>.
>>>
>>
>>
>